crates/subspace-proof-of-space/src/chiapos/table.rs

```diff
@@ -293,14 +287,14 @@ fn find_matches<'a>(
     // quotient more efficiently by subtracting base value rather than computing remainder of
     // division
     let base = (usize::from(first_right_bucket_y) / usize::from(PARAM_BC)) * usize::from(PARAM_BC);
-    for (&y, right_index) in right_bucket.ys.iter().zip(Position::ZERO..) {
```

Comment:
Pls explain this more

Reply:
We were storing indices and then adding them to the start position on line 332, instead we can start iterating from start position so we don't need to do any math later.

Reply:
Ah nice